frontend/static/js/router.js

```diff
@@ -56,4 +56,10 @@ const router = async () => {
 	view.getHTML();
 };
 
+window.addEventListener('load', () => {
+	if (!sessionStorage.getItem('isLoggedInSession')) {
```

Comment:
instead of this we could have the router make a request to the backend for the supposedly logged in user and if it get a 401, we're logged out (that happens already). I will also implement that the current page the user tried to access will be loaded after logging in. This is more of a comment on what's to come than anything for you to change :P